src/js/components/search/SearchSidebar.jsx

```diff
@@ -46,7 +46,7 @@ const filters = {
         'Award ID',
         'CFDA Program',
         'North American Industry Classification System (NAICS)',
-        'Product/Service Code (PSC)',
+        'Product or Service Code (PSC)',
```

Comment:
The same change was made in Kirk's PR [here](https://github.com/fedspendingtransparency/usaspending-website/commit/a77f040bde56e372d0968939e9cf8a7672977570#diff-09a23a0c96c144f8019c0aa93b6e13a2R49), but that's only gone into `dev`. If it needs to be fixed in QAT we can leave it here since there shouldn't be any conflicts with the exact same change